rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.retrievers/10_standard_retriever.yml

```diff
@@ -0,0 +1,529 @@
+setup:
+  - skip:
+      version: ' - 8.12.99'
+      reason: 'standard retriever added in 8.13'
+  - do:
+      indices.create:
+        index: animals
+        body:
+          mappings:
+            properties:
+              type:
+                type: keyword
+              name:
+                type: keyword
+              color:
+                type: keyword
+              count:
+                type: integer
+
+  - do:
+      bulk:
+        refresh: true
+        index: animals
+        body:
+          - '{"index": {"_id": 1 }}'
+          - '{"type": "domestic", "name": "cow", "color": "brown", "count": 1}'
+          - '{"index": {"_id": 2 }}'
+          - '{"type": "domestic", "name": "cow", "color": "spotted", "count": 2}'
+          - '{"index": {"_id": 3 }}'
+          - '{"type": "domestic", "name": "cow", "color": "spotted", "count": 3}'
+          - '{"index": {"_id": 4 }}'
+          - '{"type": "domestic", "name": "pig", "color": "pink", "count": 4}'
+          - '{"index": {"_id": 5 }}'
+          - '{"type": "domestic", "name": "pig", "color": "pink", "count": 5}'
+          - '{"index": {"_id": 6 }}'
+          - '{"type": "domestic", "name": "pig", "color": "spotted", "count": 6}'
+          - '{"index": {"_id": 7 }}'
+          - '{"type": "domestic", "name": "chicken", "color": "white", "count": 7}'
+          - '{"index": {"_id": 8 }}'
+          - '{"type": "domestic", "name": "chicken", "color": "brown", "count": 8}'
+          - '{"index": {"_id": 9 }}'
+          - '{"type": "domestic", "name": "chicken", "color": "spotted", "count": 9}'
+          - '{"index": {"_id": 10 }}'
+          - '{"type": "wild", "name": "coyote", "color": "gray", "count": 1}'
+          - '{"index": {"_id": 11 }}'
+          - '{"type": "wild", "name": "coyote", "color": "gray", "count": 2}'
+          - '{"index": {"_id": 12 }}'
+          - '{"type": "wild", "name": "coyote", "color": "white", "count": 3}'
+          - '{"index": {"_id": 13 }}'
+          - '{"type": "wild", "name": "rabbit", "color": "brown", "count": 4}'
+          - '{"index": {"_id": 14 }}'
+          - '{"type": "wild", "name": "rabbit", "color": "spotted", "count": 5}'
+          - '{"index": {"_id": 15 }}'
+          - '{"type": "wild", "name": "rabbit", "color": "white", "count": 6}'
+
+---
+"standard retriever basic":
+  - do:
+      search:
+        index: animals
+        body:
+          fields: [ "name", "count" ]
```

Comment:
I realize we never talked about it, but should `fields` be a part of a retriever, similar how `sort` is a part of `retriever`? Or is this wrong to think this way because `fields` is applicable for the final fetch phase?

Reply:
I think fields continue to make the most sense as a global value for the final result set because I don't know how you would resolve it in a simple way for multiple retrievers potentially.